package.json

```diff
@@ -64,6 +65,7 @@
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.49.2",
     "sha512crypt-node": "^1.0.2",
+    "sharp": "^0.33.4",
```

Comment:
Is the sharp library necessary?

Reply:
I think it was just needed for me to be able to deploy locally. I'll remove it from the dependencies,.